src/components/Summary/AlterationSummary.vue

```diff
@@ -272,6 +277,21 @@ export default class AlterationSummary extends Mixins(
     await this.scrollToTop(document.getElementById('app'))
   }
 
+  // sum of alteration fees
+  get alterationFees (): string {
+    if (this.getCurrentFees?.filingFees) {
+      return `($${(this.getCurrentFees.filingFees + this.getCurrentFees?.futureEffectiveFees).toFixed(2)} Fee)`
```

Comment:
What happens if filingFees is 0 ?
Or if futureEffectiveFees is 0 or undefined or null ? (Can this ever happen?)

Reply:
As now they can be null or number only, I'm checking against null so I know that the values where populated with values from the pay-api and not from `emptyFees`.